### PR TITLE
[Embeddingapi] Modify usecase project xml to implement atip uiautomator

### PR DESCRIPTION
**Pull request by jacky-young**
-Modify usecase project xml to implement atip uiautomator

Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: [Android]
Unit test result summary: pass 1, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-4759

### CODE REVIEW

File: usecase/usecase-embedding-android-tests/testscripts/PauseJsTimers.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Feature: Embedding api usecase tests
+    Scenario: Check xwalk pause & resume js time
+        When I launch "usecase-embedding-android-test" with "org.xwalk.embedded.api.sample" and "PauseTimersActivity" on android
+        And I register watcher "ClearInfoWindow" when "Info" click "confirm"
```
Comment (Honry) [inline usecase/usecase-embedding-android-tests/testscripts/PauseJsTimers.feature]: Please format the lines, "And" should be one space interval against "When" and "Then"

**Comment by Honry** [comment]
Please provide the UT result.

**Comment by jacky-young** [comment]
@jennycao @Honry please help to review it.

**Comment by Honry** [comment]
LGTM  @jennycao, please also review it.:)

**Comment by jennycao** [comment]
LGTM

**Comment by Honry** [comment]
@jacky-young, please drop a mail to notice Li, Jun to add this test suite into nightly testing.

**Pull request merged**